scripts/zones/The_Eldieme_Necropolis_[S]/npcs/Erlene.lua

```diff
@@ -48,40 +48,51 @@ function onTrigger(player,npc)
     local mJob = player:getMainJob();
     local onSabbatical = player:getQuestStatus(CRYSTAL_WAR,ON_SABBATICAL);
     local onSabbaticalProgress = player:getVar("OnSabbatical");
+    local downwardHelix = player:getQuestStatus(CRYSTAL_WAR, DOWNWARD_HELIX);
 
-    if (ENABLE_WOTG == 1 and ALittleKnowledge == QUEST_AVAILABLE) then

```

Comment:
ENABLE_WOTG exists with the other expansion controls so server operators can runs servers like it was 2004, or w/e.